cccatalog-api/cccatalog/settings.py

```diff
@@ -64,9 +64,7 @@
     'django.contrib.staticfiles',
     'django_cron',
     'oauth2_provider',
-    'social_django',
     'rest_framework',
-    'rest_framework_social_oauth2',
```

Comment:
why aren't we using this or another DRF specific library (it looks like https://jpadilla.github.io/django-rest-framework-oauth/ was extracted out from the DRF code and is somewhat official)

Reply:
Rest Framework Social OAuth2 is for integrating social login features of other sites (GitHub, Facebook, Google, etc). This is good for end-user login but not applicable to situations with API keys. It was included as a dependency before because we initially planned to have a user login for lists, but that feature was canned.

I used this particular library because it's [currently recommended by the DRF developers](https://www.django-rest-framework.org/api-guide/authentication/#django-oauth-toolkit) over `django-rest-framework-oauth`.

Reply:
I missed that it was the currently recommended library when I was skimming the docs, thanks.